packages/zowe-explorer/src/globals.ts

```diff
@@ -33,7 +33,7 @@ export let DS_DIR;
 export let CONFIG_PATH; // set during activate
 export let ISTHEIA: boolean = false; // set during activate
 export let LOG: imperative.Logger;
-export const COMMAND_COUNT = 103;
+export const COMMAND_COUNT = 102;
```

Comment:
The fact that we have gone down in the number of commands scares me a little bit.

I know the FTP extension was updated as part of this PR, but can we teat this with another extender's extension that has not updated their implementation yet?

@rudyflores or @JillieBeanSim, can you help us here?